src/index.html

```diff
@@ -37,7 +37,7 @@
   <div class="container-fluid">
     <nav class="navbar navbar-expand-lg row bg-blue navbar-dark">
       <div class="navbar-brand">
-        <img src="img/girlscodelincoln-nav.png" alt="Girls Code Lincoln" style="height: 30px;" />
+        <img src="img/girlscodelincoln-nav.svg" height="30" alt="Girls Code Lincoln" />
```

Comment:
Could you move this to the CSS instead of inline?

Reply:
Done